rialto_airflow/schema/reports.py

```diff
@@ -31,6 +31,37 @@ class Publications(ReportsSchemaBase):  # type: ignore
     updated_at = Column(DateTime, onupdate=utcnow())
 
 
+class PublicationsBySchool(ReportsSchemaBase):  # type: ignore
+    __tablename__ = "publications_by_school"
+
+    id = Column(Integer, primary_key=True, autoincrement=True)
+    doi = Column(String)
+    pub_year = Column(Integer)
+    apc = Column(Integer)
+    open_access = Column(String)
+    types = Column(String)
+    federally_funded = Column(Boolean)
+    academic_council_authored = Column(Boolean)
+    faculty_authored = Column(Boolean)
+    primary_school = Column(String)
+
+
+class PublicationsByDepartment(ReportsSchemaBase):  # type: ignore
+    __tablename__ = "publications_by_department"
+
+    id = Column(Integer, primary_key=True, autoincrement=True)
+    doi = Column(String)
+    pub_year = Column(Integer)
+    apc = Column(Integer)
+    open_access = Column(String)
+    types = Column(String)
+    federally_funded = Column(Boolean)
+    academic_council_authored = Column(Boolean)
+    faculty_authored = Column(Boolean)
+    primary_school = Column(String)
+    primary_department = Column(String)
```

Comment:
I'm seeing there are some other fields for this table requested in the Shared Table Requirements, but the logic for finding those is not in the field definitions in those requirements. These include `first_author_name, journal, keywords, open_access, pages`. I think they are not required as filters for the dashboard and are there for data download purposes, but don't want to lose track of this. If it's not ticketed, we should add a follow-on ticket.

Reply:
We do have these in the existing CSV so there should be logic for them?